contracts/vaults/STETHVault/base/RibbonVault.sol

```diff
@@ -186,35 +198,40 @@ contract RibbonVault is
         address _feeRecipient,
         uint256 _managementFee,
         uint256 _performanceFee,
-        string memory tokenName,
-        string memory tokenSymbol,
+        string memory _tokenName,
+        string memory _tokenSymbol,
         Vault.VaultParams calldata _vaultParams
     ) internal initializer {
-        VaultLifecycleSTETH.verifyConstructorParams(
+        VaultLifecycle.verifyInitializerParams(
             _owner,
             _keeper,
             _feeRecipient,
             _performanceFee,
-            tokenName,
-            tokenSymbol,
+            _managementFee,
+            _tokenName,
+            _tokenSymbol,
             _vaultParams
         );
 
         __ReentrancyGuard_init();
-        __ERC20_init(tokenName, tokenSymbol);
+        __ERC20_init(_tokenName, _tokenSymbol);
         __Ownable_init();
         transferOwnership(_owner);
 
         keeper = _keeper;
 
         feeRecipient = _feeRecipient;
         performanceFee = _performanceFee;
-        managementFee = _managementFee.mul(10**6).div(WEEKS_PER_YEAR);
+        managementFee = _managementFee.mul(Vault.FEE_MULTIPLIER).div(
+            WEEKS_PER_YEAR
+        );
         vaultParams = _vaultParams;
 
         collateralToken = IWSTETH(0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0);
```

Comment:
Since we are hardcoding it this way, we might as well make it a constant or immutable